package.json

```diff
@@ -39,7 +39,8 @@
     "react-native-webview": "^10.3.2",
     "react-navigation": "^4.1.0",
     "react-navigation-stack": "^1.10.3",
-    "sentry-expo": "^2.0.3"
+    "sentry-expo": "^2.0.3",
+    "expo-updates": "~0.0.1-rc.0"
```

Comment:
Why is it on this weird release candidate?

Reply:
hmm..i think it's still a beta version but it should be the latest cos i ran `expo install expo-updates` again and it results in the same version